### DISCUSSION TITLE
What is the way to stop listening?

### DISCUSSION THREAD

**Discussion by Vitorinox** [spatie/twitter-streaming-api#35] [Q&A]
I'm curious how can I stop listening, that is, break the connection and reload the rules

**Comment by felixdorn** [comment]
I didn't want to break anything regarding the public API of the package.

So, you can use the underlying [package](https://github.com/redwebcreation/twitter-stream-api) where there is an easy way to stop listening.

Feel free to send a PR that adds such functionality, it should be pretty trivial but I'm pretty busy currently.
